src/views/services-details/ServicesDetails.vue

```diff
@@ -19,6 +19,15 @@
           />
         </div>
       </div>
+      <es-banner-as-footer-mobile
+        :fitness="this.$router.currentRoute.params.type === 'fitness'"
```

Comment:
Instead of sending 8 props, why not sending the param type and handle the logic in the component?

Reply:
Make sense. I'll try to re-factor.